content/blog/optimize-tailwindcss-in-angular.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Optimize Tailwind CSS in Angular for deployment
+description: Remove unused Tailwind CSS utilities from your Angular production build for improved performance
+published: true
+publishedAt: 2020-12-11T17:20:00.000Z
+updatedAt: 2020-12-11T17:20:00.000Z
+tags:
+  - Angular
+  - Tailwind CSS
+  - Scully
+keywords:
+  - CSS
+  - Utility-First CSS
+authors:
+  - Marc Stammerjohann
+github: https://github.com/notiz-dev/angular-scully-tailwindcss
+---
+
+[Tailwind CSS](https://tailwindcss.com) generates a **large amount** of utility classes for your development build by design. For your Angular application you want the best performance by **only** including the classes you are actually using in your production build. Tailwind got you covered! [PurgeCSS](https://purgecss.com/) is build-in making it easy to remove unused Tailwind styles for your application.
```

Comment:
By design, [Tailwind CSS](https://tailwindcss.com) generates a **large amount** of utility classes for your development build.